### PR TITLE
Feature: Implement flatpickr replacing jquery datepicker.

### PR DESCRIPTION
**Pull request by johnhout**
I removed the old jQuery datepicker by replacing it with the newer dependency free flatpickr.

### CODE REVIEW

**Comment by ceesvanegmond** [comment]
@johnhout Could you check what happens when you empty the field in the CMS? What should we do then? Set null? Set empty?

**Comment by johnhout** [comment]
Good point, Will check thank you

**Comment by johnhout** [comment]
@ceesvanegmond New update, Set a default date on the attribute so that the datepicker can never be empty.

**Comment by ceesvanegmond** [comment]
@johnhout So there's always a date? Should the date/datetime not be nullable?

**Comment by ceesvanegmond** [comment]
👍 

@Emroni

File: public/js/main.js [hunk 1]
```diff
@@ -52,11 +52,13 @@ $(function() {
 		});
 	});
 
-	$('input.datepicker').datepicker({
-		dateFormat: "dd-mm-yy"
+	flatpickr('input.datepicker', {
+		dateFormat: 'd-m-Y',
+		enableTime: false,
+		timeFormat: "H:i",
+		time_24hr: true,
```
Comment (Emroni) [inline public/js/main.js]: Can you remove the last comma? Older browser cant handle those (both arrays and objects).

File: resources/views/attributes/DateTimeAttribute.blade.php [hunk 2]
```diff
@@ -1,16 +1,37 @@
-<label class="form-group js-placeholder" for="{{ $name }}">
+<style>
+    .datepicker input {
+        width: 40%;
+    }
+    .datepicker .btn {
+        border: 1px solid #dadada;
+        border-left: none;
+        font-size:1.12em;
```
Comment (Emroni) [inline resources/views/attributes/DateTimeAttribute.blade.php]: Since we use 16 as the base font size, please use 1.125em to get 18px (18 / 16 = 1.125).

**Comment by ceesvanegmond** [comment]
@johnhout Looks good 👍 , we should think of a way to get rid of the <style> block.

**Pull request merged**